server/handler.go

```diff
@@ -569,10 +569,12 @@ func rowToSQL(ctx *sql.Context, s sql.Schema, row sql.Row) ([]sqltypes.Value, er
 			o[i] = sqltypes.NULL
 			continue
 		}
-
-		o[i], err = s[i].Type.SQL(ctx, nil, v)
-		if err != nil {
-			return nil, err
+		// TODO: need to make sure the schema is not null as some plan schema is defined as null (e.g. IfElseBlock)
```

Comment:
Do these need the `TODO` prefix? I couldn't figure out what was left to be done. It seems like we are checking if the schema is nil, so sounded like that TODO has been done? 

The comment explaining why is good though. Interesting to know that sometimes the schema can be null, such as for IfElseBlock.